CHANGELOG.yaml

```
@@ -12,6 +12,21 @@ meta:
 
 changes:
 
+  - title: Add endpoint to shorten URLs
+    issues:
+      - https://github.com/databiosphere/azul/issues/66
+    upgrade:
+      - terraform
+      - deploy
+      - clients
+    notes: >
+           The `/url` endpoint was added to accept URLs and return a URL redirecting to the given URL.
+           Documentation:
+           https://github.com/DataBiosphere/azul/blob/2ab5c91d13e576a0e831298681e221ab0048bbcd/lambdas/service/app.py#L701
```

This link won't work, unfortunately, because it points to the commit on the feature branch, not `develop` where it will ultimately end up. This is why I am looking forward to generating the changelog from the commit history where we could dynamically generate these links on the fly.

Just do

```suggestion
           /lambdas/service/app.py#L701
```

or leave this out and file it as a separate PR after this PR has landed. I think the former is more pragmatic.